### DISCUSSION TITLE
Hide tooltip after delay?

### DISCUSSION THREAD

**Discussion by viveleroi** [ReactTooltip/react-tooltip#1161] [Q&A] [answered]
I noticed there's a delayHide prop but that delays hiding. I am looking for a way to hide the tooltip after 10 seconds, regardless of where the mouse is. I assume I could fake it by using the imperative api and the close method - is there anything I'm missing or should I file a feature request

**Comment by devmildfire** [comment]
I upvote that. It will be really nice if the tooltip has the ability to disappear even without any user interaction, just by a timeout. I use it on the "copy link" element and even with openEvents and closeEvents i can't make it just go away after initial click.

**Reply by gabrieljablonski** [reply]
#1212 is open as a feature request for this.

The issue also has a comment with this [code sample](https://codesandbox.io/p/sandbox/tooltip-forked-5385t9?file=%2Fsrc%2FApp.tsx) which shows how to implement this feature "manually", until we make it a prop for the tooltip.

**Comment by danielbarion** [comment, marked as answer]
It will be implemented on V6 based on the discussion with Gabriel in the issue.
